tests/unit/amo/test_withExperiment.js

```diff
@@ -11,6 +10,8 @@ import {
   getVariant,
   withExperiment,
 } from 'amo/withExperiment';
+// eslint-disable-next-line import/namespace
```

Comment:
I don't really understand what that rule does and why we need to disable it but I don't really mind :)

Reply:
The message from `eslint` is `No exported names found in module 'config/default'`